src/main/app/documents/documentsClient.ts

```diff
@@ -21,6 +21,10 @@ export class DocumentsClient {
     return this.getPDF(claimExternalId, 'defendantResponseReceipt', bearerToken)
   }
 
+  getClaimantHearingRequirementPDF (claimExternalId: string, bearerToken: string): Promise<Buffer> {
+    return this.getPDF(claimExternalId, 'claimantHearingRequirement', bearerToken)
```

Comment:
I can see all backend document names( 'claimantHearingRequirement' ) for the new URLs are capital letters, is it something applies here?

Reply:
As discussed, currently leaving it as how other types are done. It can be a separate PR if document type is changed.